repose-aggregator/core/repose-core/src/main/java/org/openrepose/powerfilter/PowerFilter.java

```diff
@@ -24,6 +24,8 @@
 import io.opentracing.Span;
 import io.opentracing.SpanContext;
 import io.opentracing.Tracer;
+import io.opentracing.noop.NoopSpan;
+import io.opentracing.noop.NoopSpanContext;
```

Comment:
I'm not seeing any reference to these imports. Are they vestigial?

Reply:
Yep. These were left over from the `NoOp` optimizations I removed. They're gone now.